tutorials/functional-full-stack/purescript-bridge/tutorial.md

```diff
@@ -268,7 +267,7 @@ All I want is to copy this backend data to the frontend, where both use an equiv
 There has to be a way to do that automatically.
 
 Well, there is a way to do that. Can you see the `Generic` instances we have introduced somehow?
-We are going to leverage these instances for us. In fact, I have not invented this, the [Argonaut](https://github.com/purescript-contrib/purescript-argonaut) guys did it!
+We are going to take advantage of these instances. In fact, I have not invented this - the [Argonaut](https://github.com/purescript-contrib/purescript-argonaut) guys did it!
```

Comment:
Good question. If you know for a fact that all of the people on the team were men, I think it's ok to say "guys" - otherwise we should probably make it more gender-neutral.